Author: Luca Boccassi <bluca@debian.org>
Description: disable SWIOTLB with kernel 4.16+
 Since Linux version 4.16 (4bd89ed39) swiotlb_map_sg_attrs is no longer
 exported and thus cannot be used by kernel modules.
 Disable it in the build, as it is apparently not really needed and there seems
 to be no way around it for now.
--- a/common/inc/nv-linux.h
+++ b/common/inc/nv-linux.h
@@ -1235,7 +1235,7 @@
     /* AArch64 exports these symbols directly */
     swiotlb_in_use = ((ops == &noncoherent_swiotlb_dma_ops) ||
                       (ops == &coherent_swiotlb_dma_ops));
-      #else
+      #elif LINUX_VERSION_CODE < KERNEL_VERSION(4, 16, 0)
     swiotlb_in_use = (ops->map_sg == swiotlb_map_sg_attrs);
       #endif
     #else
